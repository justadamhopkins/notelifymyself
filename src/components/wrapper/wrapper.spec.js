import React from 'react'
import { mount } from 'enzyme'
import { Wrapper } from '.'

describe('<Wrapper/>', () => {
  it('Should render component', () => {
    const wrapper = mount(<Wrapper/>)
    expect(wrapper).toMatchSnapshot()
  })
})
