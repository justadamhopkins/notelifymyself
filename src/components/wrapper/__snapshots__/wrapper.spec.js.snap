// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> Should render component 1`] = `
ReactWrapper {
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "getWrappingComponentRenderer": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <main
        class="Wrapper close"
      />,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
        "className": "Wrapper close",
      },
      "ref": null,
      "rendered": Array [
        undefined,
      ],
      "type": "main",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <main
          class="Wrapper close"
        />,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "Wrapper close",
        },
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "main",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__linkedRoots__): Array [],
  Symbol(enzyme.__unrendered__): <Wrapper />,
  Symbol(enzyme.__updatedBy__): null,
}
`;
